explorer/src/components/WorldMap.tsx

```diff
@@ -1,111 +1,80 @@
-import React, { useEffect, useState } from 'react';
-import { geoEqualEarth, geoPath } from 'd3-geo';
-import { feature } from 'topojson-client';
-import { Feature, FeatureCollection, Geometry } from 'geojson';
-// import './styles.css';
+import React from 'react';
+import { scaleLinear } from 'd3-scale';
+import { ComposableMap, Geographies, Geography } from 'react-simple-maps';
+import ReactTooltip from 'react-tooltip';
+import { Box } from '@mui/material';
+import { MainContext } from '../context/main';
+import { countriesData } from '../data/countriesData';
+import { ContentCard } from './ContentCard';
 
-const uuid = require('react-uuid');
-
-const cx = 400;
-const cy = 150;
+const geoUrl =
+  'https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json';
 
 export const WorldMap: React.FC = () => {
-  const [geographies, setGeographies] = useState<
-    [] | Array<Feature<Geometry | null>>
-  >([]);
-
-  const [scale, setScale] = useState<number>(200);
-  const [windowResizing, setWindowResizing] = useState<boolean>(false);
-
-  const projection = geoEqualEarth()
-    .scale(scale)
-    .translate([cx, cy])
-    .rotate([0, 0]);
-
-  const fetchGeographicData = (): void => {
-    fetch('/data/world-110m.json').then((response) => {
-      if (response.status !== 200) {
-        console.log('Houston, we have a problem');
-        return;
-      }
-      response.json().then((worldData) => {
-        const mapFeatures: Array<Feature<Geometry | null>> = (
-          feature(
-            worldData,
-            worldData.objects.countries,
-          ) as unknown as FeatureCollection
-        ).features;
-        setGeographies(mapFeatures);
-      });
-    });
-  };
-
-  const renderMap = () => {
-    console.log('render the map...');
-  };
-
-  useEffect(() => {
-    fetchGeographicData();
-    renderMap();
-  }, []);
-
-  useEffect(() => {
-    let timeout: any;
-    const handleResize = () => {
-      clearTimeout(timeout);
-
-      setWindowResizing(true);
-
-      timeout = setTimeout(() => {
-        setWindowResizing(false);
-      }, 200);
-    };
-    window.addEventListener('resize', handleResize);
-
-    return () => window.removeEventListener('resize', handleResize);
+  const [tooltipContent, setTooltipContent] = React.useState<string>('');
```

Comment:
```suggestion
const [tooltipContent, setTooltipContent] = React.useState<string>(null);
```

or


```suggestion
const [tooltipContent, setTooltipContent] = React.useState<string>(undefined);
```

Reply:
I have played with the MUI tooltip but a) it doesnt pair out of the box the way these two play with each other nicely. b) may just be something simple like style override / position / zIndex - but can we use RTT for now and once PR is in i'm happy to explore the MUI one?